logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/UserSessionScope.kt

```diff
@@ -341,6 +357,16 @@ abstract class UserSessionScopeCommon(
             mlsClientProvider,
             client.deregisterNativePushToken
         )
+    private val featureConfigRepository: FeatureConfigRepository
+        get() = FeatureConfigDataSource(featureConfigApi = authenticatedDataSourceSet.authenticatedNetworkContainer.featureConfigApi)
+    val isFileSharingEnabled: IsFileSharingEnabledUseCase get() = IsFileSharingEnabledUseCaseImpl(userConfigRepository)
+
+    val getRemoteFileSharingStatusAndPersistUseCase: GetRemoteFileSharingStatusAndPersistUseCase
```

Comment:
Reviewing the Reloaded PR, I realized about this tiny change that we can do, so we are on the same page with naming as the rest of the code =)
```suggestion
    val getRemoteFileSharingStatusAndPersist: GetRemoteFileSharingStatusAndPersistUseCase
```